pkg/server/server.go

```diff
@@ -432,13 +435,14 @@ func (s *ProxyServer) serveRecvFrontend(stream client.ProxyService_ProxyServer,
 	for pkt := range recvCh {
 		switch pkt.Type {
 		case client.PacketType_DIAL_REQ:
-			klog.V(5).Infoln("Received DIAL_REQ")
 			random := pkt.GetDialRequest().Random
+			address := pkt.GetDialRequest().Address
+			klog.V(3).InfoS("Received DIAL_REQ", "dialID", random, "address", address)
```

Comment:
I think this is the same value as "dialAddress" we have in labels (Eg. https://github.com/kubernetes-sigs/apiserver-network-proxy/blob/master/pkg/agent/client.go#L440). It would be good to keep the naming consistent.